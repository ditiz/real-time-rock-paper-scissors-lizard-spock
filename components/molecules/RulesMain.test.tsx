import "@testing-library/jest-dom"
import { render, screen } from "@testing-library/react"
import RulesMainMolecule from "./RulesMain"

it("renders RulesMainMolecule unchanged", () => {
  const { container } = render(<RulesMainMolecule />)
  expect(container).toMatchSnapshot()
})

it("shoud contain game description", () => {
  render(<RulesMainMolecule />)
  expect(
    screen.getByText(
      /The game is an expansion on the game Rock, Paper, Scissors/
    )
  ).toBeInTheDocument()
})
