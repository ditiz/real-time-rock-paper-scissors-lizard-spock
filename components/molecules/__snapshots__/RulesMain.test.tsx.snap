// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders RulesMainMolecule unchanged 1`] = `
<div>
  <p
    class="text"
  >
    The game is an expansion on the game Rock, Paper, Scissors. Each player picks a variable and reveals it at the same time. The winner is the one who defeats the others. In a tie, the process is repeated until a winner is found.
  </p>
</div>
`;
